application/common/config/main.php

```diff
@@ -1,22 +1,20 @@
 <?php
 
+use common\components\SesMailer;
 use Sil\JsonLog\target\JsonStreamTarget;
 use Sil\Log\EmailTarget;
 use Sil\PhpEnv\Env;
 use yii\db\Connection;
 use yii\helpers\Json;
-use yii\swiftmailer\Mailer;
+use yii\swiftmailer\Mailer as SwiftMailer;
```

Comment:
Swiftmailer is deprecated. So, you might consider refactoring this to use symfony.

Reply:
Added this as a new ticket (IDP-785).